modules/shortcodes/googlemaps.php

```diff
@@ -124,7 +124,9 @@ function jetpack_googlemaps_shortcode( $atts ) {
 			}
 		}
 
-		return '<div class="' . esc_attr( $css_class ) . '"><iframe width="' . $width . '" height="' . $height . '" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="' . $url . '"></iframe></div>';
+		$sandbox = class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ? 'sandbox="allow-popups allow-scripts allow-same-origin"' : '';
+
+		return '<div class="' . esc_attr( $css_class ) . '"><iframe width="' . $width . '" height="' . $height . '" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="' . $url . '" ' . $sandbox . '></iframe></div>';
```

Comment:
Should we take that opportunity to clean this up, escape everything, and make it more readable?

```diff
diff --git a/modules/shortcodes/googlemaps.php b/modules/shortcodes/googlemaps.php
index 6171b2241..c64ff8196 100644
--- a/modules/shortcodes/googlemaps.php
+++ b/modules/shortcodes/googlemaps.php
@@ -124,9 +124,20 @@ function jetpack_googlemaps_shortcode( $atts ) {
 			}
 		}
 
-		$sandbox = class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request() ? 'sandbox="allow-popups allow-scripts allow-same-origin"' : '';
-
-		return '<div class="' . esc_attr( $css_class ) . '"><iframe width="' . $width . '" height="' . $height . '" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" src="' . $url . '" ' . $sandbox . '></iframe></div>';
+		$sandbox = class_exists( 'Jetpack_AMP_Support' ) && Jetpack_AMP_Support::is_amp_request()
+			? 'sandbox="allow-popups allow-scripts allow-same-origin"'
+			: '';
+
+		return sprintf(
+			'<div class="%1$s">
+				<iframe width="%2$d" height="%3$d" frameborder="0" scrolling="no" marginheight="0" marginwidth="0" %5$s src="%4$s"></iframe>
+			</div>',
+			esc_attr( $css_class ),
+			absint( $width ),
+			absint( $height ),
+			esc_url( $url ),
+			esc_attr( $sandbox )
+		);
 	}
 }
 add_shortcode( 'googlemaps', 'jetpack_googlemaps_shortcode' );

```

Reply:
Yeah, that's a good idea. I'll commit that later tonight if that's alright.

Reply:
It's applied in https://github.com/Automattic/jetpack/pull/15317/commits/7aae669ff32ceb23268adcffba7b08a902e291e9